### DISCUSSION TITLE
"Error: Request failed with status code 500" when trying to deploy postgres or mysql

### DISCUSSION THREAD

**Discussion by silasabbott** [caprover/caprover#1988] [Q&A] [answered]
I'm on a fresh install of CapRover running on a Raspberry Pi. The error occurs after the "Configuring" stage and immediately when the "Deploying" stage begins. This seems to be happening for me on all one-click apps that have a database. I haven't had this in other hosting environments so I'm sure I'm doing something wrong here! Any ideas?

**Comment by githubsaturn** [comment]
500 error means that your server failed. You can view the logs and see what happened:
```
docker service logs captain-captain --since 5m
```

**Reply by silasabbott** [reply]
Thanks! Turned out to be a permissions error with /var/run/docker.sock

**Comment by silasabbott** [comment, marked as answer]
Turned out to be a permissions issue with `/var/run/docker.sock`, probably because I was managing from a non-root user.

I determined this by viewing logs:
```bash
docker service logs captain-captain --since 5m
```

And I fixed it by running: 
```bash
sudo chmod 666 /var/run/docker.sock
```
